CHANGELOG-3.5.md

```diff
@@ -102,6 +102,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 - [Refactor consistentindex](https://github.com/etcd-io/etcd/pull/11699).
 - [Add log when etcdserver failed to apply command](https://github.com/etcd-io/etcd/pull/11670).
 - Improve [count-only range performance](https://github.com/etcd-io/etcd/pull/11771).
+- Improve [start-up performance](https://github.com/etcd-io/etcd/pull/11779).
```

Comment:
Can we make this more descriptive (e.g. remove redundant storage restore operation)?

Reply:
updated. thanks.